relay-general/src/protocol/user_report.rs

```diff
@@ -8,9 +8,15 @@ pub struct UserReport {
     /// The event ID for which this user feedback is created.
     pub event_id: EventId,
     /// The user's name.
+    // Empty strings are in fact stored in the database already
```

Comment:
You can remove those comments.

Should this be `Option<String>`, instead, to support explicit `null`?

Reply:
I'm just reading the sourcecode of Sentry, but here's what I believe holds:

The "schema" for user report is "defined" is as the set of possible keyword-arguments to the Python model itself. Anything goes that the model constructor accepts. Explicit null was never really supported. Also changing to Option means that we would have to coerce to empty string on serialization precisely to avoid running `UserReport(email=None).save()` which would violate NotNullConstraint of the database.

Reply:
Sure, yet, we're designing the ingestion interface here. To keep it consistent with the main event schema, and to not invert expectations, I would prefer to allow `null` going forward, too.

Reply:
To add to the above, the API can and must abstract over how database access is or used to be handled in Sentry. This may change any time, and now that it occurs in a separate code base, we're no longer bound by implicit behavior that improperly tested code used to expose accidentally.